eclair-core/src/main/scala/fr/acinq/eclair/db/sqlite/SqliteAuditDb.scala

```diff
@@ -282,11 +319,14 @@ class SqliteAuditDb(sqlite: Connection) extends AuditDb with Logging {
             case Some(RelayedPart(_, _, _, "channel", timestamp)) => incoming.zip(outgoing).map {
               case (in, out) => ChannelPaymentRelayed(in.amount, out.amount, paymentHash, in.channelId, out.channelId, timestamp)
             }
-            case Some(RelayedPart(_, _, _, "trampoline", timestamp)) => TrampolinePaymentRelayed(paymentHash, incoming, outgoing, timestamp) :: Nil
+            case Some(RelayedPart(_, _, _, "trampoline", timestamp)) =>
+              val (nextTrampolineAmount, nextTrampolineNodeId) = trampolineByHash.getOrElse(paymentHash, (0 msat, PublicKey(hex"020000000000000000000000000000000000000000000000000000000000000000")))
```

Comment:
nit: there's a `PlaceHolderPubKey` that you can use in `Transactions.scala`, you can either import it here or use `PrivateKey(ByteVector32.One).publicKey` directly if you prefer

Reply:
Done.